### DISCUSSION TITLE
how to change message width ?

### DISCUSSION THREAD

**Discussion by tendertree** [folke/noice.nvim#659] [Q&A]
<img width="1217" alt="스크린샷 2023-11-23 234141" src="https://github.com/folke/noice.nvim/assets/3841867/00d9d36e-7257-4a6c-95e4-991f45e0bc4d">

even I change width of message but it seemed not to work :(
I try to set 

```
	messages = {
		width = 10,
		max_width = 10,        -- max number of columns to display for a message
	},
```

or add notify, popupmenu, but not to work

**Comment by ilias777** [comment]
You can see here &rarr; #750 

You need to set the max_width value in the nvim-notify options. Look [here](https://github.com/rcarriga/nvim-notify/blob/d333b6f167900f6d9d42a59005d82919830626bf/doc/nvim-notify.txt#L189) in the docs.
